assets/js/components/sell-quick-start.component.js

```diff
@@ -181,18 +185,15 @@ function sellQuickStartController ($scope, $rootScope, currency, buySell, Alerts
     $scope.getQuote();
   };
 
+  $scope.numLabels = () => Wallet.my.wallet.labels._accounts.length;
```

Comment:
If you're trying to know if the user has more than one account, maybe something like `$scope.hasMultipleAccounts = () => Wallet.accounts().length > 1` would be better?